manifests/worker.pp

```diff
@@ -171,4 +179,12 @@
     ensure => $docker_ensure,
     dns    => $advertise_addr,
   }
+
+  if $xylem_backend {
```

Comment:
`undef` is falsey.. but I guess we should be consistent with `if $var {...` vs `if $var != undef`.. not sure if you have a preference?